tests/lambda_functions/delivery_status_processor_lambda/test_delivery_status_processor_lambda.py

```diff
@@ -0,0 +1,239 @@
+    monkeypatch.setenv("LOG_LEVEL", "DEBUG")
+    monkeypatch.setenv("CELERY_TASK_NAME", "CELERY_TASK_NAME")
+    monkeypatch.setenv("ROUTING_KEY", "ROUTING_KEY")
+
+
+LAMBDA_MODULE = "lambda_functions.delivery_status_processor_lambda.delivery_status_processor_lambda"
+
+
+@pytest.fixture(scope="function")
+def event():
+    """Generates a sample ALB received Twilio delivery status event object."""
+    return {
+        'requestContext': {
+            'elb': {
+                'targetGroupArn': '<DEV TARGET GROUP>'
+            }
+        },
+        'httpMethod': 'POST',
+        'path': '/deliverystatus',
+        'queryStringParameters': {},
+        'headers': {
+                'accept': '*/*',
+                'connection': 'close',
+                'content-length': '227',
+                'content-type': 'application/x-www-form-urlencoded; charset=utf-8',
+                'host': 'dev-api.va.gov',
+                'i-twilio-idempotency-token': '50609cf4-07f3-4e3f-ac42-044bd13bbc6c',
+                'user-agent': 'TwilioProxy/1.1',
+                'x-amzn-trace-id': 'Self=<SOME VALUE>',
+                'x-forwarded-for': '<COMMA SEPARTED IPS>',
+                'x-forwarded-host': 'dev-api.va.gov:443',
+                'x-forwarded-port': '443',
+                'x-forwarded-proto': 'https',
+                'x-forwarded-scheme': 'https',
+                'x-home-region': 'us1',
+                'x-real-ip': '<SOME IP>',
+                'x-twilio-signature': '<SOME VALUE>'
+        },
+        'body': 'U21zU2lkPXRoaXNpc3NvbWVzbXNpZCZTbXNTdGF0dXM9c2VudCZNZXNzYWdlU3RhdHVzPXNlbnQmVG89JTJCMTExMTExMTExMTEmTWVzc2FnZVNpZD1zb21lbWVzc2FnZWlkZW50aWZpZXImQWNjb3VudFNpZD10d2lsaW9hY2NvdW50c2lkJkZyb209JTJCMjIyMjIyMjIyMiZBcGlWZXJzaW9uPTIwMTAtMDQtMDE=',
+        'isBase64Encoded': True
+    }
+
+
+def test_sys_exit_with_unset_queue_env_var(monkeypatch, all_path_env_param_set):
+    monkeypatch.delenv("DELIVERY_STATUS_RESULT_TASK_QUEUE")
+
+    with pytest.raises(SystemExit):
+        from lambda_functions.delivery_status_processor_lambda.delivery_status_processor_lambda import (
+            delivery_status_processor_lambda_handler
+        )
+
+
+def test_sys_exit_with_unset_deadletter_queue_env_var(monkeypatch, all_path_env_param_set):
+    monkeypatch.delenv("DELIVERY_STATUS_RESULT_TASK_QUEUE_DEAD_LETTER")
+
+    with pytest.raises(SystemExit):
+        from lambda_functions.delivery_status_processor_lambda.delivery_status_processor_lambda import (
+            delivery_status_processor_lambda_handler
+        )
+
+
+def test_invalid_event_event_none(mocker, all_path_env_param_set):
+    from lambda_functions.delivery_status_processor_lambda.delivery_status_processor_lambda import (
+        delivery_status_processor_lambda_handler
+    )
+
+    sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_sqs')
+
+    # Test a event is None
+    delivery_status_processor_lambda_handler(None, None)
+
+    sqs_mock.assert_called_once()
```

Comment:
event of `None` should raise. This does not check for that. In event: None or a full even it will call SQS.